mass_mailing_list_dynamic/tests/test_dynamic_lists.py

```diff
@@ -2,19 +2,19 @@
 # Copyright 2020 Hibou Corp. - Jared Kipe
 # Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
-
 from mock import patch
 
 from odoo.exceptions import ValidationError
 from odoo.tests import common
 
 
-@common.at_install(False)
-@common.post_install(True)
 class DynamicListCase(common.SavepointCase):
+    at_install = False
```

Comment:
This doesn't serve since v13, so it can be removed if this is working, as no need of deferring it.